core/scripts/sql/texera_ddl.sql

```diff
@@ -19,16 +19,20 @@ SET GLOBAL time_zone = '+00:00'; # this line is mandatory
 
 CREATE TABLE IF NOT EXISTS user
 (
-    `name`      VARCHAR(32)                 NOT NULL,
-    `uid`       INT UNSIGNED AUTO_INCREMENT NOT NULL,
-    `password`  VARCHAR(256),
-    `google_id` VARCHAR(256) UNIQUE,
-    PRIMARY KEY (`uid`),
+    `email` varchar(256) NOT NULL,
+	`uid` int unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
the primary key is not on the first column, this is not the usual convention. I know this is a legacy issue. Not sure if we want to take this chance to make `uid` the first column of this table? minor suggestion only.

Reply:
I was confused, too. I will fix it.